src/utils/redirect.ts

```diff
@@ -6,7 +6,7 @@ import { UserType } from '@/types/interfaces'
 export const redirectIfTaskCta = (searchParams: Record<string, string>) => {
   const taskId = z.string().safeParse(searchParams.taskId)
   if (taskId.data) {
-    redirect(`${apiUrl}/detail/${taskId.data}/iu?token=${z.string().parse(searchParams.token)}`)
+    redirect(`${apiUrl}/detail/${taskId.data}/iu?token=${z.string().parse(searchParams.token)}&isRedirect=true`)
```

Comment:
what happens if parse fails here @rrojan ?

Reply:
We `console.error` a ZodError in logs and user gets a properly buillt error response, which is handled by `withErrorHandler` HOF in `src/app/core/utils/withErrorHandler.ts`

Reply:
But this case should almost never happen, since this function is triggered by an iframe path by Copilot platform and the platform always sends a token with every request

Reply:
so you mean if parse fails zod throws an exception. thanks for explanation.